src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double height;
+    private final double width;
+
+    public Rectangle(String color, double length, double width) {
```

Comment:
The parameter 'length' should be named 'height' to match the field and avoid confusion.